src/devices/controller/controller.cpp

```diff
@@ -28,14 +31,16 @@ ControllerMode mode = ControllerMode::Effect;
 ControllerMode prev_mode = mode;
 
 // Pin definitions
-constexpr int kSwitchLeft = PA0;
-constexpr int kSwitchRight = PA1;
-constexpr int kSwitchBottom = PA2;
+constexpr std::array<uint8_t, 3> kLeftButtons = {PB2, PB14, PB15};
```

Comment:
These are directly wired to pins now?

Reply:
Yes, there were enough free pins for this.